.openshiftio/booster.yaml

```diff
@@ -1,2 +1,7 @@
 name: WildFly Swarm - HTTP
 description: Runs a WildFly Swarm HTTP application
+versions:
```

Comment:
@kenfinnigan @Ladicek What terms should we use here? community/redhat was mentioned on the ML, but I don't know if this has been decided.